packages/datx-jsonapi-angular/package.json

```diff
@@ -1,101 +1,37 @@
 {
   "name": "@datx/jsonapi-angular",
   "version": "2.3.1",
-  "description": "DatX mixin for Angular JSON API support",
```

Comment:
We're missing this whole part in package.json, especially the main and module parts.

Reply:
I've missed re-adding these back in the package.json that angular-cli created from scratch. I will re-add those extra properties that were in the old package.json.